code/game/objects/structures/morgue.dm

```diff
@@ -264,9 +264,9 @@
 	else if (src.locked == 0)
 		playsound(src.loc, 'sound/items/Deconstruct.ogg', 50, 1)
 		src.connected = new /obj/structure/c_tray( src.loc )
-		step(src.connected, SOUTH)
+		step(src.connected, dir)

```

Comment:
That hardcodes the crematorium tray to only ever open facing southwards, regardless of the direction of the actual crematorium thing. It really messes things up. The morgue was changed like four years past, but the crematorium never had the same matching change. The whole file is an utter mess, but that puts it in a slightly more working state.